example_projects/register_demo/requirements.txt

```diff
@@ -0,0 +1,3 @@
+starlette
+uvicorn
+piccolo[postgres]
```

Comment:
@dantownsend Please add `httpx` and `python-multipart` as dependecies in example project.

Reply:
Will do, thanks